File: naive/src/fib.erl
-module(fib).
-export([main/0]).

fib(N) when N < 3 -> 1;
fib(N) when N > 2 -> fib(N-1) + fib(N-2).

main() ->
    {ok, Arg} = io:read(""),
    io:format("res: ~p\n", [fib(Arg)]).

File: naive/src/fib.ml
let rec fib n =
    match n with
    | n when n < 3 -> 1
    | _ -> fib (n - 1) + fib (n - 2)
;;

let num = Sys.argv.(1)
let n = int_of_string num
let res = fib n
let () = Printf.printf "res = %d" res
